docs/conf.py

```diff
@@ -315,6 +315,8 @@ def linkcode_resolve(domain, info):
     return None
   if not info['fullname']:
     return None
+  if info['module'].split(".")[0] != 'jax':
+     return None
```

Comment:
@jakevdp — I needed to add this here to remove the broken "source" link from that page. WDYT?

Reply:
Looks good!